contributor-docs/versioning.md

```diff
@@ -132,6 +134,67 @@ In this situation, changing the layout of the `button` to `flex` would collapse
 the whitespace present between the text "Save" and "changes" and would be
 considered a breaking change.
 
+### A component changes its usage of a CSS Custom Property
+
+semver bump: potentially **major**
+
+When a component changes how it uses a CSS Custom Property, the change could be
+considered breaking if the CSS Custom Property is a part of the component's
+public API or participates in a broader system for design tokens. The breakage
```

Comment:
> if the CSS Custom Property is a part of the component's public API

Do we have examples of components where we intentionally provide CSS Custom Properties as part of the component's public API? I can't think of a time where I've used a CSS Custom Property as part of the component API.

Reply:
@mperrotti same, I can't think of a time either.